src/test/common/ios/simulatorPlist.test.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {SimulatorPlist} from "../../../common/ios/simulatorPlist";
+
+import * as assert from "assert";
+import * as path from "path";
+import * as Q from "q";
+import * as sinon from "sinon";
+
+suite("plistBuddy", function() {
+    suite("commonContext", function() {
+        test("Should correctly find the NSUserDefaults plist file for the simulator", function() {
+            const projectRoot = path.join("project", "root");
+
+            const bundleId = "com.contoso.app";
+
+            const findSimulatorHomeCommand = "xcrun simctl getenv booted HOME";
+            // The emulator's home folder is /simulator/home
+            const findSimulatorHomeResult = path.join("/", "simulator", "home");
+
+            const prefix = path.join("Containers", "Data", "Application");
+            const suffix = path.join("Library", "Preferences");
+
+            // The emulator has 3 apps, app1 app2 and app3
+            const appIds = ["app1", "app2", "app3"];

```

Comment:
I'd use some valid GUIDs in the amount that you normally find in a simulator here.